docker/lr-pb/environment.yml

```diff
@@ -4,8 +4,13 @@ channels:
   - conda-forge
```

Comment:
another general comment: let's pin down all versions of involving "environment.yml".
Creating another ticket for this. No action needed here.